functions/__fzf_search_current_dir.fish

```diff
@@ -6,8 +6,8 @@ function __fzf_search_current_dir --description "Search the current directory. R
     set fd_opts --color=always $fzf_fd_opts
     set fzf_arguments --multi --ansi $fzf_dir_opts
     set token (commandline --current-token)
-    # expand ~ in the directory name since fd can't expand it
-    set expanded_token (string replace --regex -- "^~/" $HOME/ $token)
+    ​# expand the token (which may include tilde, variables, etc.) into full path
+    set expanded_token (eval echo -- \"$token\")
```

Comment:
What's an example where removing the quotes would break this?

Reply:
I'm just quoting it like in `__fzf_preview_file`.

Reply:
I just can't think of when token when ever be separated by an unescaped space. Whatever, I suppose it's safer this way.

Reply:
[You](https://github.com/PatrickF1/fzf.fish/pull/137#issuecomment-801632385) are totally right... I was too careful to do it right.

Reply:
Oh shoot, this is actually a bug. Quoting it broke this for tilde. Just tested.

Reply:
Fixed in https://github.com/PatrickF1/fzf.fish/commit/83cb78fbb27dd969d20daace826de9c78216067d